### ISSUE TITLE
Actions are not working when with a `force_locale` params

### ISSUE THREAD

**Issue by Annedj** [avo-hq/avo#1771]
### Describe the bug

Hi Adrian 👋 

We just noticed an issue with `Actions` when on a different locale. We have added a `Publish` to our Event resource, but when on a different locale we are getting an error as the path given includes the `force_locale` param:

<img width="1552" alt="Screenshot 2023-05-30 at 10 45 07" src="https://github.com/avo-hq/avo/assets/57643651/ca3323aa-f071-4cff-b312-899e65f8e8dd">

### Steps to Reproduce
Steps to reproduce the behaviour:

- Go on a resource with a `force_locale`
- Click on an Action
- The action `show` method goes through but it breaks when submitting the [action form](https://github.com/avo-hq/avo/blob/main/app/views/avo/actions/show.html.erb#L10)

### Expected behavior & Actual behavior

`@resource.records_path` on this [line](https://github.com/avo-hq/avo/blob/main/app/views/avo/actions/show.html.erb#L12) shouldn't include the `force_locale` params:

```ruby
@resource.records_path
=> "/v4/admin/resources/local_events?force_locale=fr"
```

### System configuration
**Avo version**:
2.33.1

**Rails version**:
7.0

**Ruby version**:
3.2.2

**License type**:

 - [ ] Community
 - [X] Pro

**Are you using Avo monkey patches or overriding views or view components?**

 - [ ] Yes. If so, please post code samples.
 - [X] No

#### Impact

 - [ ] High impact (It makes my app un-usable.)
 - [X] Medium impact (I'm annoyed, but I'll live.) Users cannot use actions in their locale and the actions policy/permissions are often dependent on locales (so users don't edit a locale they don't have access to)
 - [ ] Low impact (It's really a tiny thing that I could live with.)

#### Urgency

 - [ ] High urgency (I can't continue development without it.)
 - [ ] Medium urgency (I found a workaround, but I'd love to have it fixed.)
 - [ ] Low urgency (It can wait. I just wanted you to know about it.)

**Comment by adrianthedev** [comment]
This is fixed and released in Avo [2.33.2](https://github.com/avo-hq/avo/releases/tag/v2.33.2)

**Comment by Annedj** [comment]
Thank you so much @adrianthedev!
